codespell_lib/data/dictionary.txt

```diff
@@ -30622,6 +30622,12 @@ specifiction->specification
 specifictions->specifications
 specificy->specify, specificity, specifically,
 specifid->specified
+specifiec->specific
+specifiecally->specifically
+specifiecation->specification
+specifiecations->specifications
+specifiecd->specified
+specifieced->specified
```

Comment:
```text
specifiecs->specifics
```
* https://github.com/search?q=specifiecs&type=code

Reply:
Ah, indeed. Nice catch and thanks a lot for this suggestion.

Added with 71e42c21f304c63013d590135bbebbf4829e8c57